addons/hr_holidays/views/hr_leave_views.xml

```diff
@@ -109,11 +109,10 @@
                         <div class="oe_kanban_global_click container">
                             <div class="row no-gutters">
                                 <div class="col-2">
-                                    <img t-att-src="kanban_image('hr.employee', 'image_64', record.employee_id.raw_value)"
+                                    <img t-att-src="kanban_image('hr.employee', 'image_128', record.employee_id.raw_value)"
                                         t-att-title="record.employee_id.value"
                                         t-att-alt="record.employee_id.value"
-                                        width="40" height="40"
-                                        class="oe_kanban_avatar float-left mr4 col-md-"/>
+                                        class="oe_kanban_avatar o_image_40 float-left mr4"/>
```

Comment:
Same

Reply:
Not `.o_kanban_record_bottom .oe_kanban_bottom_right`